Makefile

```diff
@@ -8,11 +8,17 @@ DOCKER := ENVKIND=$(ENVKIND) docker-compose -f $(COMPOSE_FILE)
 DOCKER_RUN := $(DOCKER) run --rm
 
 clean:
-	@find . -name '*.pyc' -exec -delete
+	@python setup.py clean
+	@find $(MAKEFILE_DIR) -name '*.pyc' -type f -delete
+	@rm -rf $(MAKEFILE_DIR)/build $(MAKEFILE_DIR)/dist \
+	    $(find $(MAKEFILE_DIR) -name __pycache__ -type d)
 
 develop: clean
 	@python setup.py develop
 
+typecheck:
```

Comment:
Coool!

Reply:
No. I will follow up with a couple PRs to get it passing.